docs/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+tocMaxDepth: 2
+---
+
+import { WideTile, Flex } from '@redocly/developer-portal/ui';
+
+# Redocly CLI
+
+Redocly CLI is an open source command-line tool for working with OpenAPI descriptions, developer portals, and other API lifecycle operations including API linting, enhancement, and bundling.
+
+<div>
+  <Flex justifyContent="space-between" flexWrap="wrap">
+    <WideTile to="../redoc/deployment/cli.md" header="Redocly and Redoc">
+      Redoc is our open source API documentation tool. Use `redocly` (this project) to work with your Redoc documentation projects.
+    </WideTile>
+    <WideTile to="./file-management.md" header="Manage OpenAPI descriptions">
+      Split an OpenAPI description into logical chunks, bundle the chunks to create a single file, or even join existing definitions into one.
+    </WideTile>
+    <WideTile to="./decorators.md" header="Transform OpenAPI descriptions">
+      Don't have access to the source code? Use decorators to add, change, or remove content from generated OpenAPI descriptions.
```

Comment:
I'd have preferred to have kept the copy from the previous `index.md` file, unless for some reason it won't fit here.

Reply:
I didn't use the same content because in some bullets there were multiple links and one bullet had no links. For instance, the first bullet on the current page has a link to the Redoc GitHub repo and a link to the API docs page in the Redocly CLI docs. And the last bullet - "Seamless integration with Redocly" - does not include any links. I could add the content as is with multiple links and remove the link from the tile itself if you would prefer.